scripts/src/main/resources/scripts/functions

```diff
@@ -1155,32 +1162,19 @@ function doIsPackageJsonContainingScript() {
   fi
 }
 
-# Checks if Windows Subsystem for Linux(WSL) is enabled or not.
-function doIsWSLEnabled() {
-  if doIsWindows
-  then
-    wsl_distro_list=$(wsl -l)
-    if [[ $wsl_distro_list != *"no installed distributions"* ]];
-    then
-      return
-    else
-      doEcho "WSL is disabled or No Linux distributions found in WSL. Please enable WSL and install any linux distribution to proceed with docker setup."
-      doEcho "Refer here to setup WSL. https://docs.microsoft.com/en-us/windows/wsl/install-win10"
-      return 255
-    fi
-  else
-    return 255  
-  fi 
-}
-
-# Checks if docker is installed on Windows Subsystem for Linux(WSL)
-function doIsDockerInstalledOnWSL() {
-  if wsl command -v docker &> /dev/null
+# Ensurese that Windows WSL is properly available.
+function doRequireWsl() {
+  local error
+  if ! command -v wsl &> /dev/null
+  then 
+    error="WSL 2 is not installed.\nPlease install WSL 2."
+  elif wsl -l > /dev/null
   then
     return
-  else   
-    return 255  
-  fi   
+  else
+    error="WSL 2 is disabled or no linux distributions found in WSL.\nPlease ensure WSL is enabled and a proper linux distribution is installed in WSL."
+  fi
+  doFail "${error}\nTo fix this follow these instructions:\nhttps://docs.microsoft.com/en-us/windows/wsl/install-win10"
```

Comment:
I think it would be better to use the short link here. The content will be displayed in the user's language and the link does not beak when MS restructures the docs. https://aka.ms/wsl2-install

Reply:
@SchettlerKoehler thanks for this feedback.
However, this PR is about the MacOS support so I will not include further windows changes here and better get this merged now.
For windows I will create another PR immediately after merge but there I will entirely remove this and go to Docker Desktop instead what makes that link to MS obsolete. In case we should decide against this PR and Docker Desktop for Windows we should do what you suggested. You can also edit the URL in the browser on github and create a PR from that with 2 clicks if you fear that this change might be forgotten or lost.